core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -137,7 +138,13 @@ private[spark] class EventLoggingListener(
     // scalastyle:on println
     if (flushLogger) {
       writer.foreach(_.flush())
-      hadoopDataStream.foreach(_.hflush())
+      hadoopDataStream.foreach(ds => {
```

Comment:
OK, if in doubt, would it be perhaps safer to preserve the existing behavior and hflush in all cases?